vulnfeeds/cves/versions.go

```diff
@@ -613,22 +660,37 @@ func Commit(u string) (string, error) {
 	return "", fmt.Errorf("Commit(): unsupported URL: %s", u)
 }
 
-// For URLs referencing commits in supported Git repository hosts, return a GitCommit.
-func extractGitCommit(link string) *GitCommit {
+// For URLs referencing commits in supported Git repository hosts, return an AffectedCommit.
+// TODO(andrewpollock): do something better aligned with the OSV schema definition once https://github.com/ossf/osv-schema/issues/187
+func extractGitCommit(link string, commitType string) (ac AffectedCommit, err error) {
+	if !slices.Contains([]string{"Introduced", "LastAffected", "Limit", "Fixed"}, commitType) {
```

Comment:
Let's change all these `commitType` to the same enum used on vulns.go:418 by moving it out of the function.

Reply:
Done in https://github.com/google/osv.dev/pull/1510/commits/e397536f57d7cee950e44093ed3b2470d2d78763